File: ios/RCTMGL-v10/RCTMGLMarkerViewManager.m
#import "React/RCTBridgeModule.h"
#import <React/RCTViewManager.h>
#import <Foundation/Foundation.h>

@interface RCT_EXTERN_REMAP_MODULE(RCTMGLMarkerView, RCTMGLMarkerViewManager, RCTViewManager)

RCT_EXPORT_VIEW_PROPERTY(coordinate, NSString)

@end

File: ios/RCTMGL-v10/RCTMGLMarkerViewManager.swift
import Foundation
import MapboxMaps

@objc(RCTMGLMarkerViewManager)
class RCTMGLMarkerViewManager : RCTViewManager {
  @objc
  override static func requiresMainQueueSetup() -> Bool {
    return true
  }
  
  override func view() -> UIView! {
    return RCTMGLMarkerView()
  }
}
